vnext/Desktop.UnitTests/MemoryMappedBufferTests.cpp

```diff
@@ -0,0 +1,149 @@
+#include "MemoryMappedBuffer.h"
+#include "utilities.h"
+
+#include <CppUnitTest.h>
+
+#include <shlwapi.h>
+#include <windows.h>
+
+#include <cstring>
+#include <memory>
+
+using facebook::jsi::Buffer;
+using facebook::jsi::JSINativeException;
+using facebook::react::utilities::checkedReinterpretCast;
+using Microsoft::JSI::makeMemoryMappedBuffer;
+using Microsoft::VisualStudio::CppUnitTestFramework::Assert;
+
+namespace {
+
+static constexpr const wchar_t *const testFileName = L"temp.txt";
```

Comment:
>testFileName [](start = 38, length = 12)

this should really be writing to temp directory instead of directory where process is executing

Reply:
Changed to using a temp file in the temp directory.

---
In reply to: [320913312](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320913312) [](ancestors = 320913312)